packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -197,6 +202,10 @@ export class MonacoEditor implements TextEditor, IEditorReference {
         return this.onFocusChangedEmitter.event;
     }
 
+    isSuggestVisible(): boolean {
+        return SuggestState.Idle !== this.suggestController._model.state;
```

Comment:
`this.editor.getContribution('editor.contrib.suggestController')._widget.suggestWidgetVisible.get()` -  it is exactly the same as `suggestWidgetVisible` when context in VS Code

Reply:
I have seen that. I do not know why would it be better to get this information from the widget with a string-based ID, but I do not mind going with this approach either.

Reply:
Both ways work for me as well. I commented it because of https://github.com/theia-ide/theia/pull/1706#discussion_r181582056 to reduce amount of dependencies on internals.